examples/databases/postgres/devbox.json

```diff
@@ -1,8 +1,8 @@
 {
   "packages": [
-    "postgresql"
+    "postgresql@latest"
```

Comment:
I will be surprised this works in testscripts in our CICD without needing to set PGHOST explicitly to a short directory

Reply:
(not this line specifically, I meant more generally that i'll be surprised the postgres example passes.

Reply:
You are right! I need to overwrite it.